pilot/model/validation.go

```diff
@@ -873,6 +876,12 @@ func ValidateEgressRule(msg proto.Message) error {
 		if err := ValidateEgressRulePort(port); err != nil {
 			errs = multierror.Append(errs, err)
 		}
+
+		if cidrDestinationService && Protocol(strings.ToUpper(port.Protocol)) != ProtocolTCP {
```

Comment:
Do you think it might make sense to use a protobuf enum for the port protocol for regress rules?
https://github.com/istio/api/blob/master/proxy/v1/config/egress_rule.proto#L52

Reply:
@ijsnellf Good point, let me fix that in a separate PR, I opened an issue https://github.com/istio/istio/issues/1664